lms/djangoapps/api_manager/users/views.py

```diff
@@ -1012,6 +1009,13 @@ def get(self, request, user_id, course_id):
         if len(queryset):
             current_grade = queryset[0].grade
             proforma_grade = queryset[0].proforma_grade
+            progress_summary = json.loads(queryset[0].progress_summary)
+            grade_summary = json.loads(queryset[0].grade_summary)
+            grading_policy = json.loads(queryset[0].grading_policy)
+        else:
```

Comment:
I'm not really seeing the scenario that would require this else statement, although I feel like it can't hurt to include as long as we have the proper test coverage.  If we are unable to find a record in the gradebook, we should also write a new gradebook entry, rather than just returning the generated values.  That way we will avoid doing the work the next time around.